Tools/MiniBrowser/mac/SettingsController.m

```diff
@@ -672,6 +677,16 @@ - (BOOL)appleColorFilterEnabled
     return [[NSUserDefaults standardUserDefaults] boolForKey:AppleColorFilterEnabledPreferenceKey];
 }
 
+- (void)toggleSiteSpecificQuirksModeEnabled:(id)sender
+{
+    [self _toggleBooleanDefault:SiteSpecificQuirksModeEnabledPreferenceKey];
+}
+
+- (BOOL)siteSpecificQuirksModeEnabled
```

Comment:
Whoops meant to comment here.

Won't this default to Quirks disabled on first launch since the default key is missing?

Reply:
`NSArray *onByDefaultPrefs` covers this

Reply:
Nice I missed that line.